components/test/publishSubscribe.js

```diff
@@ -55,7 +55,7 @@ var receivedError: NatsTypescriptTemplateError | undefined = undefined;
 var receivedMsg: ${subscribeClientClass}.${getMessageType(message)} | undefined = undefined;
 ${receivedVariableDeclaration}
 
-var publishMessage: ${publishClientClass}.${getMessageType(message)} = ${publishMessageExample};
+var publishMessage: ${publishClientClass}.${getMessageType(message)} = ${publishClientClass}.${getMessageType(message)}.unmarshal(${publishMessageExample});

```

Comment:
@magicmatatjahu all the tests uses the unmarshal function together with the openapi-example generator. So that handles the initialization of objects.